app/scripts/constants/sentry-state.ts

```diff
@@ -42,6 +42,11 @@ export const SENTRY_BACKGROUND_STATE = {
   },
   AuthenticationController: {
     isSignedIn: false,
+    sessionData: {
+      profile: true,
+      accessToken: false,
+      expiresIn: true,
+    },
```

Comment:
This will expose some useful information when inspecting sentry errors related to authenticated services. Exposes:
```
profile.profileId - the uuid for a given profile
profile.identifierId - a unique hash from what a user used to sign in with
expiresIn - a string that you can parse with `Date.parse()`. Can be useful if to determine if an access token expired.
```

Reply:
Image to show what we are exposing.

![Screenshot 2025-01-06 at 11 28 23](https://github.com/user-attachments/assets/85ea2ff9-d386-4c3e-9209-a913bb87db3d)